datafusion/core/tests/sqllogictests/test_files/array.slt

```diff
@@ -357,3 +357,33 @@ query ?
 select make_array(x, y) from foo2;
 ----
 [1.0, 1]
+
+statement ok
+CREATE EXTERNAL TABLE fixed_size_list_array STORED AS PARQUET LOCATION 'tests/data/fixed_size_list_array.parquet';
+
+query ?
+select * from fixed_size_list_array;
```

Comment:
Could you also add a query that is like `arrow_typeof(f0)` to show in the test what the type of the column is?